assets/reports/markdown.template

```diff
@@ -0,0 +1,21 @@
+# osde2e weather report
+
+## Report Information
+This report was generated on {{.ReportDate}}
```

Comment:
I feel like this section is redundant. It could just go as the first line under Summary OR do something like `OSDe2e Weather Report {{.ReportDate}}`

Reply:
Like this idea.